sdk_contrib/typed_span/src/main/java/io/opentelemetry/sdk/contrib/typedspan/http/HttpClientSpanWrapper.java

```diff
@@ -0,0 +1,31 @@
+package io.opentelemetry.sdk.contrib.typedspan.http;
+
+import static io.opentelemetry.sdk.contrib.typedspan.http.HttpSpanWrapperBuilder.STATUS_CODE_KEY;
+import static io.opentelemetry.sdk.contrib.typedspan.http.HttpSpanWrapperBuilder.STATUS_TEXT_KEY;
+
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Status;
+
+public class HttpClientSpanWrapper extends HttpSpanWrapper {
+  public HttpClientSpanWrapper(Span span) {
+    super(span);
+  }
+
+  public HttpClientSpanWrapper setStatusCode(HttpStatusCode status) {
+    this.getRawSpan().setAttribute(STATUS_CODE_KEY, status.getCode());
```

Comment:
This is missing the span status (https://github.com/open-telemetry/opentelemetry-specification/blob/ac75cfea2243ac46232cbc05c595bb0c018e2b58/specification/data-http.md#status).